website/content/docs/upgrading/index.mdx

```diff
@@ -68,20 +68,27 @@ Perform these steps on each standby:
 2. Replace the Vault binary with the new version
 3. Start the standby node
 4. Unseal the standby node
+5. Verify `vault status` shows correct Version and HA Mode is `standby`
+6. Review the node's logs to ensure successful startup and unseal
 
 At this point all standby nodes will be upgraded and ready to take over. The
 upgrade will not be complete until one of the upgraded standby nodes takes over
 active duty. To do this:
 
-1. Properly shut down the remaining (active) node. Note: it is _**very
-   important**_ that you shut the node down properly. This causes the HA lock to
-   be released, allowing a standby node to take over with a very short delay.
+1. Properly shut down the remaining (active) node 
+
+   ~> **Note:** It is important that you shut the node down properly. 
+   This will perform a step-down and release the HA lock, allowing a standby
+   node to take over with a very short delay.
    If you kill Vault without letting it release the lock, a standby node will
    not be able to take over until the lock's timeout period has expired. This
    is backend-specific but could be ten seconds or more.
+   
 2. Replace the Vault binary with the new version; ensure that `mlock()` capability is added to the new binary with [setcap](/docs/configuration#disable_mlock)
```

Comment:
Should this mlock note be added above on the standby steps?

Reply:
Yep - added